tests/framework-tests/core/js/FluidJSTests.js

```diff
@@ -230,6 +230,53 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         });
     });
 
+    fluid.tests.roundToDecimalTests = [
+        {num: 1.555, scale: 5, expected: 1.555},
+        {num: 1.555, scale: 4, expected: 1.555},
+        {num: 1.555, scale: 3, expected: 1.555},
+        {num: 1.555, scale: 2, expected: 1.56},
+        {num: 1.555, scale: 1, expected: 1.6},
+        {num: 1.555, scale: 1.3, expected: 1.6},
+        {num: 1.555, scale: 1.5, expected: 1.56},
+        {num: -1.555, scale: 5, expected: -1.555},
+        {num: -1.555, scale: 4, expected: -1.555},
+        {num: -1.555, scale: 3, expected: -1.555},
+        {num: -1.555, scale: 2, expected: -1.56},
+        {num: -1.555, scale: 1, expected: -1.6},
+        {num: 1.555, scale: 0, expected: 2},
+        {num: 1.555, scale: -2, expected: 2},
+        {num: 1.555, scale: NaN, expected: 2},
+        {num: 1.555, scale: undefined, expected: 2},
+        {num: 1.555, scale: null, expected: 2},
+        {num: 1.555, scale: "two", expected: 2},
+        {num: 1.005, scale: 2, expected: 1.01},
+        {num: -1.005, scale: 2, expected: -1.01},
+        {num: 55.549, scale: 1, expected: 55.5},
+        {num: -55.549, scale: 1, expected: -55.5},
+        {num: 54.9, scale: 0, expected: 55},
+        {num: -54.9, scale: 0, expected: -55},
+        {num: 55.55, scale: 1, expected: 55.6},
+        {num: -55.55, scale: 1, expected: -55.6},
+        {num: 55.551, scale: 1, expected: 55.6},
+        {num: -55.551, scale: 1, expected: -55.6},
+        {num: 55.59, scale: 1, expected: 55.6},
+        {num: -55.59, scale: 1, expected: -55.6},
+        // IEEE 754 binary floating point numbers treat 0.014999999999999999, 0.0150000000000000001 and 0.015 as equivalent.
+        {num: 0.014999999999999999, scale: 2, expected: 0.02},
+        {num: -0.014999999999999999, scale: 2, expected: -0.02},
+        {num: 0.0150000000000000001, scale: 2, expected: 0.02},
+        {num: -0.0150000000000000001, scale: 2, expected: -0.02},
+        {num: 0.015, scale: 2, expected: 0.02},
+        {num: -0.015, scale: 2, expected: -0.02}
+    ];
```

Comment:
I looked into adding a test for this, but it's not working and I'm not sure what to do about it. 

```javascript
{num: 1.005 - 1, scale: 2, expected: 0.01}
```

So, 1.005 - 1 should equal 0.005 however, it's returning 0.004999999999999893 instead. Probably in relation to the floating point precision. With that in mind, if I look at what it should really equate to mathematically I'd expect the algorithm to round up to 0.01. However, if I look at the number that is being calculated, I'd expect it to round down to 0.